packages/adapter-node/index.js

```diff
@@ -36,8 +48,51 @@ export default function ({ out = 'build' } = {}) {
 			await utils.prerender({
 				dest: `${out}/prerendered`
 			});
+			if (precompress) {
+				await compress(`${out}/prerendered`);
+			}
 		}
 	};
 
 	return adapter;
 }
+
+/**
+ * @param {string} directory
+ */
+async function compress(directory) {
+	const files = await fg('*.{html,js,json,css,svg,xml}', {
+		cwd: directory,
+		dot: true,
+		absolute: true,
+		baseNameMatch: true
+	});
+
+	for await (const file of files) {
```

Comment:
This should be one Promise.all call, instead of the sequential stepper.

Reply:
Changed.

My reason behind this was to prevent to many files open at the same time.

Should we use something like [p-map](https://www.npmjs.com/package/p-map) to limit the number of concurrent file operations?

Reply:
Done.